services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -807,6 +825,40 @@ private long buildAndinstallMeter(final IOFSwitch sw, final DatapathId dpid, fin
         return pushFlow(sw, "--InstallMeter--", meterMod);
     }
 
+    @VisibleForTesting
+    long installMeterForDefaultRule(IOFSwitch sw, long cookie, boolean isBroadcast) throws OfInstallException {
+        OFFactory ofFactory = sw.getOFFactory();
+
+        if (isSwitchSupportsMeters(sw) && isSwitchSupportsPktpsFlag(sw)) {
```

Comment:
Lack of `isSwitchSupportsPktpsFlag()` support is not enough reason to not install meter on rule that direct packets into controller.

We can recalculate meter limit on kbps ... lets take packet size as 500 bytes so 1000pkts is (500 * 1000 / 1024 kbps)... so do not use pkts for switches that do not support it (we should dig further why they don't support it, according to spec they should support it).

(critical)